cmake/RippledSettings.cmake

```diff
@@ -18,7 +18,7 @@ if(tests)
   endif()
 endif()
 
-option(unity "Creates a build using UNITY support in cmake. This is the default" ON)
+option(unity "Creates a build using UNITY support in cmake." OFF)
```

Comment:
The commit message here is a little confusing.
> Unity is disabled in conanfile.py, fix cmake for consistency 

It makes me think you've disabled unity entirely. How about:
> Default to non-unity builds in cmake to match conanfile.py 

I'm not asking you to change the commit message in the PR, but when the final PR is merged, consider using something like this for that commit message.